lib/gym/project.rb

```diff
@@ -83,6 +83,7 @@ def build_settings(key: nil, optional: true)
         command = "xcrun xcodebuild -showBuildSettings #{BuildCommandGenerator.project_path_array.join(' ')}"
         Helper.log.info command.yellow unless Gym.config[:silent]
         @build_settings = `#{command}`
+        Helper.log.info @build_settings if $verbose
```

Comment:
That's too much information, did you take a look at this? We can't just dump it onto the terminal, that's why it shows the command before so the user can run the command manually :+1: